src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -156,7 +150,7 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
     // A model to compute token embeddings.
     // Input shape: [N, conversation length].
     // Output shape: [1, conversation length, hidden_size].
-    ov::InferRequest m_embedding;
+    std::shared_ptr<EmbeddingsModel> m_embedding;
```

Comment:
Why is it a ptr instead of regular member? There's no pImpl

Reply:
fixed